cortex_m0_mpy121666.s

```diff
@@ -0,0 +1,199 @@
+// Implementation of multiplication of an fe25519 bit value with the curve constant 121666.

```

Comment:
This file is a duplicate of the one in the `crypto` folder (and in any case shouldn't be in the top-level directory). Shoul be removed